.circleci/config.yml

```diff
@@ -53,6 +53,23 @@ executors:
     - image: *ci_support_image
 
 commands:
+  install-tcell:
+    steps:
+      - run:
+          name: Download tcell files from bucket
+          command: |
+            vault read -format=json  secret/pepper/prod/v1/conf | jq -r .data.gcp.serviceKey | gcloud auth activate-service-account --key-file=-
+            TAR_FILE_URL=`gsutil ls $TCELL_PATH  | sort -r | head -1`
+
+            if [ -z $TAR_FILE_URL ]
+            then
+              echo "Could not find tcell archive ${TCELL_PATH}"
+              exit 1
+            fi
+            TAR_FILE_PATH="/tmp/$(basename -- $TAR_FILE_URL)"
+            gsutil cp  $TAR_FILE_URL $TAR_FILE_PATH
+
+            tar --no-same-owner -xvf $TAR_FILE_PATH
```

Comment:
Mind adding a comment that you expect this command will create a directory called tcell ? Had to figure where the `tcell` directory was coming from in deploy and concluded it had to be from here.